src/kakarot/accounts/eoa/externally_owned_account.cairo

```diff
@@ -80,9 +74,27 @@ func __execute__{
     calldata_len: felt,
     calldata: felt*,
 ) -> (response_len: felt, response: felt*) {
-    let (response: felt*) = alloc();
-    // TODO: parse, call kakarot, and format response
-    return (response_len=0, response=response);
+    alloc_locals;
+    let (tx_info) = get_tx_info();
+    let version = tx_info.version;
+    with_attr error_message("ExternallyOwnedAccount: deprecated tx version: {version}") {
+        assert_le(1, version);
```

Comment:
is this > or >= version 1?

Reply:
good question I don't know this comes from @Flydexo ; I don't now if we really need this actually

Reply:
This comes from the Openzeppelin code, didn't dig more about it